dbt/src/main/kotlin/com/getstrm/pace/dbt/ManifestParser.kt

```diff
@@ -40,6 +40,7 @@ object ManifestParser {
         return modelNodes.filter {
             it.isPaceEnabled()
         }.map {
+            System.err.println("Creating policy for model ${it.originalFilePath}")
```

Comment:
Why `err`?

Reply:
Because I didn't want to interfere with the current stdout. Let's dicsuss.